rtengine/iptransform.cc

```diff
@@ -382,21 +442,23 @@ static float calcGradientFactor(const struct grad_params& gp, int x, int y)
             float val = ((float)(gy - gp.top_edge_0) * gp.ys_inv);
 
             if (gp.bright_top) {
-                val = 1.0 - val;
+                val = 1.f - val;
             }
 
-            if (gp.scale < 1.0) {
-                val = pow(xsinf(val * M_PI / 2), 3);
+            val *= (M_PI / 2);
```

Comment:
My `math.h` has a `M_PI_2` define. Plus, there is a `RT_PI` in this file, which could be copied as `RT_PIF` to get rid of the `double`. But maybe not worth the effort...